libtsuba/include/katana/TxnContext.h

```diff
@@ -85,9 +92,11 @@ class KATANA_EXPORT TxnContext {
 
   bool GetTopologyWrite() const { return topology_write_; }
 
+  bool GetCommitManifest() const { return commit_manifest_; }
+
 private:
-  std::string ConcatRDGProperty(std::string rdg_dir, std::string prop) {
-    return rdg_dir + kPropSeparator + prop;
+  inline std::string ConcatRDGProperty(std::string rdg_dir, std::string prop) {
+    return Uri::JoinPath(rdg_dir, prop);
```

Comment:
Why do you need an inline function that does the same thing as `JoinPath`?

Reply:
I thought it's more versatile if in the future we want to change e.g. the way to concatenate. I've change it to use `JoinPath` directly now.